src/components/TinyRouter.vue

```diff
@@ -1,213 +1,240 @@
+    TinyRouterInstance.push(path)
+  })
 }
 
 export default TinyRouter
 
-export { TinyRouter }
+export {TinyRouter}
 
 /**
  * Vue plugin installer - adds TinyRouter component and $router global property
  * Usage: app.use(TinyRouterInstall)
  */
 export const TinyRouterInstall = {
-	install( app ) {
-		app.component( 'TinyRouter', TinyRouter )
-		app.config.globalProperties.$router = {
-			/** Navigate to a route - this.$router.push('/path') */
-			push( path ) { TinyRouterInstance?.push( path ) },
-			
-			/** Get current route path - this.$router.route */
-			get route() { return TinyRouterInstance?.route },
-			
-			/** Get current component instance - this.$router.component */
-			get component() { return TinyRouterInstance?.currentComponent },
-			
-			/** Get route parameters object - this.$router.params.id */
-			get params() { return TinyRouterInstance?.routeParams },
-			// this is native history API so doesnt need to be wrapped, dont work in memorymode
-			// go( n ) { history.go( n ) },
-			// forward() { history.forward() },
-			// back() { history.back() }
-		}
-	}
+  install(app) {
+    app.component('TinyRouter', TinyRouter)
+    app.config.globalProperties.$router = {
+      /** Navigate to a route - this.$router.push('/path') */
+      push(path) {
+        TinyRouterInstance?.push(path)
+      },
+
+      /** Get current route path - this.$router.route */
+      get route() {
+        return TinyRouterInstance?.route
+      },
+
+      /** Get current component instance - this.$router.component */
+      get component() {
+        return TinyRouterInstance?.currentComponent
+      },
+
+      /** Get route parameters object - this.$router.params.id */
+      get params() {
+        return TinyRouterInstance?.routeParams
+      },
+      // this is native history API so doesnt need to be wrapped, dont work in memorymode
+      // go( n ) { history.go( n ) },
+      // forward() { history.forward() },
+      // back() { history.back() }
+    }
+  }
 }
 
 /**
  * Programmatic navigation and state for Composition API
  */
-export const useRouter = () => ( {
-    push( path ) { TinyRouterInstance?.push( path ) },
-    get route() { return routeState.value.route },
-    get params() { return routeState.value.params },
-    get component() { return TinyRouterInstance?.currentComponent }
-} )
+export const useRouter = () => ({
+  push(path) {
+    TinyRouterInstance?.push(path)
+  },
+  get route() {
+    return routeState.value.route
+  },
+  get params() {
+    return routeState.value.params
+  },
+  get component() {
+    return TinyRouterInstance?.currentComponent
+  }
+})
 
 /**
  * Reactive route state for Composition API
  */
-export const useRoute = () => ( {
-    get route() { return routeState.value.route },
-    get params() { return routeState.value.params }
-} )
+export const useRoute = () => ({
+  get route() {
+    return routeState.value.route
+  },
+  get params() {
+    return routeState.value.params
+  }
+})
```

Comment:
seems there were a lot of reformating  in the PR, not blocking but I will prefer to keep the files as small as possible ( also easier to read in one line when its so trivial )